src/functions/Output.ps1

```diff
@@ -773,6 +829,25 @@ function Get-WriteScreenPlugin ($Verbosity) {
     New-PluginObject @p
 }
 
+function Write-AzureDevopsLogMessage {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [string[]] $Message,
+
+        [Parameter(Mandatory)]
+        [ValidateSet('Error', 'Warning', IgnoreCase)]
+        [string]
+        $LogType
+    )
+
+    foreach ($line in $Message) {
+
+        # https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=powershell#logissue-log-an-error-or-warning
+        & $SafeCommands['Write-Host'] "##vso[task.logissue type=$LogType]$line"
+    }
```

Comment:
I think we should skip the foreach and only append to first line, typically `[-] mypath.to.failing.test`. The error flags are used in the report, so appending the special prefix to every message will fill the report with noise for the same error. Ex.
![image](https://user-images.githubusercontent.com/3436158/122450100-e9dc9400-cfa6-11eb-947c-77d28e8e9565.png)